src/common/mfu_flist_walk.c

```diff
@@ -257,7 +260,10 @@ static void walk_getdents_create(CIRCLE_handle* handle)
 
         /* stat top level item */
         struct stat st;
-        int status = mfu_lstat(path, &st);
+        int status;
+        mfu_file_t* mfu_file;
+        mfu_file = *pfile;
+        status = mfu_file_lstat(path, &st, mfu_file);
```

Comment:
Let's put the declaration on the same line as initialization for these.
```suggestion
        mfu_file_t* mfu_file = *pfile;
        int status = mfu_file_lstat(path, &st, mfu_file);
```

Reply:
Let's apply that change throughout.